### DISCUSSION TITLE
Action Not Cancel

### DISCUSSION THREAD

**Discussion by devmyung** [community/community#49039] [Actions]
### Select Topic Area

Product Feedback

### Body

액션을 실행을 시켰는데 너무 오래 걸리고
그래서 취소 시키려고 하지만 취소가 안됩니다.

Action is too long to run.
So I want cancel this work, but not cancel.

**Comment by jsoref** [comment]
There was an outage: https://www.githubstatus.com/incidents/qfby3shb6r3w

But, if you're having problems, you can use https://support.github.com/
